GoogleDataTransport/GDTCCTTests/Integration/GDTCCTIntegrationTest.m

```diff
@@ -27,21 +27,10 @@
 #import "GoogleDataTransport/GDTCCTTests/Unit/TestServer/GDTCCTTestServer.h"
 #import "GoogleDataTransport/GDTCORTests/Common/Categories/GDTCORFlatFileStorage+Testing.h"
 
-#import <nanopb/pb.h>
```

Comment:
For context, the diff in this file is to refactor out some helper methods into a utility class that is used elsewhere